Framework/DataHandling/src/LoadSwans.cpp

```diff
@@ -0,0 +1,286 @@
+#include "MantidDataHandling/LoadSwans.h"
+#include "MantidAPI/FileProperty.h"
+#include "MantidDataHandling/LoadHelper.h"
+#include "MantidGeometry/Instrument/ComponentHelper.h"
+#include "MantidKernel/StringTokenizer.h"
+
+#include <map>
+#include <iostream>
+#include <fstream>
+#include <algorithm>
+#include <boost/tokenizer.hpp>
+
+namespace Mantid {
+namespace DataHandling {
+
+using namespace Mantid::Kernel;
+using namespace Mantid::Geometry;
+using namespace Mantid::API;
+using namespace Mantid::DataObjects;
+
+// Register the algorithm into the AlgorithmFactory
+DECLARE_ALGORITHM(LoadSwans)
+
+//----------------------------------------------------------------------------------------------
+/** Constructor
+ */
+LoadSwans::LoadSwans() {
+  // m_ws = EventWorkspace_sptr(new EventWorkspace());
+  m_ws = boost::make_shared<EventWorkspace>();
```

Comment:
Either use C++11 in class initialization and delete the constructor or put this in the initializer list.